src/slg/engines/pathoclbase/compiletextures.cpp

```diff
@@ -1931,6 +1944,16 @@ string CompiledScene::GetTexturesEvaluationSourceCode() const {
 					AddTextureSourceCall(texs, "Spectrum", tex->divideTex.tex2Index));
 				break;
 			}
+
+            case slg::ocl::ROUNDING_TEX: {
+                AddTextureSource(source, "Rounding", "float", "Float", i,
+                    AddTextureSourceCall(texs, "Float", tex->roundingTex.tex1Index) + ", " +
+                    AddTextureSourceCall(texs, "Float", tex->roundingTex.tex2Index));
+                AddTextureSource(source, "Rounding", "float3", "Spectrum", i,
+                    AddTextureSourceCall(texs, "Float", tex->divideTex.tex1Index) + ", " +
+                    AddTextureSourceCall(texs, "Float", tex->divideTex.tex2Index));
```

Comment:
I think you have a copy/paste mistake here (`divideTex`)

Reply:
Even worse, I typed it in by hand wrong.